File: app/src/main/java/com/example/android/cryptoapp/rest/CryptoCurrencyService.java
package com.example.android.cryptoapp.rest;

import com.example.android.cryptoapp.currency_data.JsonResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by OZMA NIG COM LTD on 05-Oct-17.
 */

public interface  CryptoCurrencyService {

    @GET("pricemulti?fsyms=BTC,ETH")
    Call<JsonResponse> getJsonResponse(@Query("tsyms") String tsyms);

}

File: app/src/main/java/com/example/android/cryptoapp/data/source/remote/CryptoCurrencyService.kt
package com.example.android.cryptoapp.data.source.remote

import com.example.android.cryptoapp.data.model.CurrencyAbbreviation
import com.example.android.cryptoapp.data.model.CryptoRatesResponse
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by OZMA NIG COM LTD on 05-Oct-17.
 */
interface CryptoCurrencyService {
    @GET("pricemulti?fsyms=BTC,ETH")
    suspend fun getACurrencyRate(@Query("tsyms") tsyms: String?): Response<CryptoRatesResponse>
    
    @GET("pricemulti?fsyms=BTC,ETH")
    suspend fun getSpecifiedCurrencyRate(@Query("tsyms")  currencyAbbreviation : Array<out CurrencyAbbreviation?>): Response<List<CryptoRatesResponse>>
}